draft-birkholz-scitt-software-use-cases.md

```diff
@@ -85,55 +85,53 @@ deployment chain diagram here
 
 # Software Supply Chain Use Cases
 
-## Trust Bond between Package Supplier and the Signing Authority
+## Release of Package Managers
 
-A certain software component product is created and packaged by a Supplier.
-The package by itself does not include a proof of authenticity.
-A signing authority is tasked with adding a proof of authenticity.
-Trust has to be established from the Supplier towards the Signing Authority - and vice versa.
-The mutual trust relationship (trust bond) between Supplier and Signing Authority is established per each individual software component package.
+Package producers (Suppliers) release packages to distributors. The package by itself does not include a proof of authenticity. A signing authority, coming from the distributor is tasked with adding a proof of authenticity. However there is no verifiable information as to what relationship exists between the original package producer and the distributor.
+
+Often it is relied implicitly by the statement made by the distributor about the supplier.
 
 A consumer of a released software wants:
 
-* to understand and verify that an actual trust bond exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
+* to understand and verify that an actual relationship link exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
 
 There is no standardized way to:
 
-* enable the consumer to verify that a trust bond for a certain software component package exists and is still valid.
+* enable the consumer to verify that a relationship link for a certain software component package  between the supplier and the distributor(its signing authority) exists and is still valid.
+
+## Multi party Evaluation of a Released Software Component
```

Comment:
```suggestion
## Multi Stakeholder Evaluation of a Released Software Component
```

Reply:
There is no requirement that the distributor be the signing authority. The signing authority is whomever the software supplier chooses, and many cases the supplier and signing authority are the same entity. That's how it is with our SAG-PM software; REA is both supplier and signing authority.

Reply:
@rjb4standards are you available sometime tomorrow to have a discussion with me and Henk?
Also what is your email address? We want to schedule a meeting with you..

Reply:
dick@reliableenergyanalytics.com   I can meet tomorrow